cmd/bisync/testdata/test_all_changed/golden/_testdir_path1.._testdir_path2.copy1to2.que

```diff
@@ -0,0 +1,8 @@
+"RCLONE_TEST"
```

Comment:
@ncw 

Double quotes are not actually needed for most listings.
Probably we could use the more discriminating `bisync.escapePath()` here.